examples/photogrammetry/photogrammetryAPI.ipynb

```diff
@@ -0,0 +1,332 @@
+    "\n",
+    "The tag detection is done in two steps. First we use Convolution Neural Networks (CNN) for detection of tags, and thereafter optical character recognition (OCR) is used for recognizing the tag text. In the last step, the image pixel coordinates extracted by the tag detector are mapped to 3D locations.\n",
+    "\n",
+    "\n",
+    "<img src=\"images/pipeline.png\" width=\"600\" align=\"center\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step-by-step example\n",
+    "\n",
+    "The photogrammetry API is currently only enabled for the **mltest** tenant. If you want access from other tenants, please contact us via the **#ml-vision** Slack channel or send us an email."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import requests\n",
+    "from urllib.request import urlretrieve\n",
+    "\n",
+    "HEADERS = {\"api-key\": os.environ[\"COGNITE_API_KEY\"]} # Read API key as an environment variable\n",
+    "BASEURL = \"https://api.cognitedata.com/api/playground\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### List Photogrammetry models\n",
+    "\n",
+    "List all available Photogrammetry models:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def list_photogrammetry_models():\n",
+    "    url = BASEURL + \"/vision/photogrammetry\"\n",
+    "    r = requests.get(url, headers=HEADERS)\n",
+    "    return r.json()\n",
+    "\n",
+    "list_photogrammetry_models()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Add assets to your Tenant\n",
+    "\n",
+    "When contextualising the 3D model, we compare the detected tags suggested by the model, with actual asset tags obtained from either the assets in the current tenant or provided as an argument. If the current tenant does not have the asset you want to detect, you can add it using the following function:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def add_asset(name):\n",
+    "    from cognite.client import CogniteClient\n",
+    "    client = CogniteClient()\n",
+    "    my_asset = Asset(name)\n",
+    "    assets_to_post = [my_asset]\n",
+    "    res = client.assets.post_assets(assets_to_post)\n",
+    "    \n",
+    "# add_asset(\"50CX0001A\")"
```

Comment:
Should this be commented out?

Reply:
Yup is should be commented out, or removed. 

It is only there to show people how one can add a new asset to your tenant. I.e if someone create a demo tenant and would want to use one of the metal tag around the office, they would want to add the asset to their tenant. 

The two more realistic usecases would be:
1. They would use it on a real platform, so that the assets already exist in the database.
Or 2.  they use create_photogrammetry_model with the tag_list argument such that one does not need to populate the tenant with fake assets.